ct-aa/dotTheory---JavaScript
# ✨ JS-614: Setting Up a Basic Express App

# Related Tickets
- [JS-614](https://codeandtheory.atlassian.net/browse/JS-614)

# Description
Added the **Setting Up a Basic Express App** challenge solutions.

# Validation
* Open the Terminal and go to the directory of the **Setting Up a Basic Express App* folder.
* Run "node app.js".
* Visit http://localhost:3000/ on the browser.
* The browser should display "I Love Treehouse!".

<img width="1789" alt="Screen Shot 2023-01-03 at 5 57 06 PM" src="https://user-images.githubusercontent.com/93116252/210335008-c107b966-acd1-41db-a9f5-ad3e5077d536.png">

# Risks
N/A
